src/Health/HealthController.ts

```diff
@@ -3,24 +3,18 @@ import * as Pino from 'pino'
 import { pick, pipeP } from 'ramda'
 
 import { childWithFileName } from 'Helpers/Logging'
-import { IPFSHashFailure, ClaimIPFSHashPair } from 'Interfaces'
+import { HealthError, IPFSHashFailure, TransactionAnchorRetryInfo } from 'Interfaces'
 import { IPFSHashTxId } from 'Messaging/Messages'
-
-import { BlockchainInfo, WalletInfo, NetworkInfo, IPFSInfo, EstimatedSmartFeeInfo, HealthDAO } from './HealthDAO'
-import { TransactionAnchorRetryInfo, IPFSDirectoryHashDAO } from './IPFSDirectoryHashDAO'
-
+import { BlockchainInfo, EstimatedSmartFeeInfo, HealthDAO, IPFSInfo, NetworkInfo, WalletInfo } from './HealthDAO'
```

Comment:
line break above this line deleted by accident I think?

Reply:
Not by accident. Do we want a line break to separate different types of internal imports? Is it because there's a path?

Reply:
Yeah, we have been keeping a line break between project relative and file relative imports. I think most of the repos are following this

Reply:
I will fix in the next PR: https://github.com/poetapp/node/pull/901/commits/24f6572b455003e7ca86decf41bdd344b5b0b8d2